modules/dataquery/ajax/queryEqual.php

```diff
@@ -16,7 +16,10 @@
 $category = $_REQUEST['category'];
 $fieldName = $_REQUEST['field'];
 $value = $_REQUEST['value'];
-$value = is_numeric($value) ? $value : "\"$value\"";
+
+if(!is_numeric($value) && $value !== "null") {
+	$value = "\"$value\"";
+}
```

Comment:
@driusan is there a case in which a user would specifically search for the string `"null"` and not mean that the want the values which are `null`